MH4U Dex/Constants.h

```diff
@@ -0,0 +1,115 @@
+//
+//  Constants.h
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/18/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Constants : NSObject
+
+FOUNDATION_EXPORT NSString *const JSON;
+FOUNDATION_EXPORT NSString *const TrueString;
+FOUNDATION_EXPORT NSString *const LowConstString;
+FOUNDATION_EXPORT NSString *const HighConstString;
+FOUNDATION_EXPORT NSString *const GConstString;
```

Comment:
[The "Variables" section in our coding standards](https://github.com/vokal/Coding-Standards/tree/master/iOS#variables), third bullet point:

> If a variable's scope exceeds a single class it should be treated like a global variable and prefixed with the project's three letter prefix.